website/pages/docs/drivers/docker.mdx

```diff
@@ -406,6 +406,16 @@ The `docker` driver supports the following configuration in the job spec. Only
 - `readonly_rootfs` - (Optional) `true` or `false` (default). Mount
   the container's filesystem as read only.
 
+- `runtime` - (Optional) A string representing a configured runtime to pass to docker.
+  This is equivilent to the --runtime argument in the docker CLI
+  For example, to use gvisor:
+
+  ```hcl
+  config {
+    runtime = "runsc"
```

Comment:
```suggestion
    runtime = "runc"
```

Reply:
Apparently this actually does mean `runsc` - as it refers to gVisor configuration in https://gvisor.dev/docs/user_guide/quick_start/docker/

Reply:
TIL!